.github/workflows/code-coverage.yml

```diff
@@ -0,0 +1,21 @@
+# This workflow will build a Java project with Gradle
+# For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-gradle
+
+name: Push Test Coverage to SonarCloud
+
+on:
+  push:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: '11'
+      - run: chmod +x gradlew
```

Comment:
I just wondering, is the gradlew file is already executable?

Reply:
Shall we just update the CI flow for the coverage report to avoid running the test twice?